plugins/org.jkiss.dbeaver.ext.oceanbase/plugin.xml

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.2"?>
+
+<plugin>
+
+    <extension point="org.jkiss.dbeaver.dataSourceProvider">
+        <datasource
+                class="org.jkiss.dbeaver.ext.oceanbase.OceanbaseDataSourceProvider"
+                description="Oceanbase"
+                id="oceanbase"
+                parent="generic"
+                label="Oceanbase"
+                dialect="oceanbase">
+                
+            <tree
+                  icon="icons/ob_icon.png"
+                  label="OceanBase data source"
+                  path="oceanbase">
+                <folder type="org.jkiss.dbeaver.ext.oceanbase.mysql.model.OceanbaseMySQLCatalog" label="%tree.databases.node.name" icon="#folder_schema" description="%tree.databases.node.tip">
+                    <items label="%tree.database.node.name" path="database" property="catalogs" icon="#database">
+                        <folder type="org.jkiss.dbeaver.ext.mysql.model.MySQLTable" label="%tree.tables.node.name" icon="#folder_table" description="%tree.tables.node.tip">
```

Comment:
Is `MySQLTable` supposed to be there? Did you mean to use `OceanbaseTable` here, since you have such class?

Reply:
Yes, I'm using OceanbaseTable instead. I missed it before